specification/src/main/asciidoc/platform/Profiles.adoc

```diff
@@ -188,54 +188,50 @@ Technologies]”.
 
 The following technologies are required:
 
-* EJB 3.2 (except for EJB entity beans and
-associated EJB QL, which have been made optional)
-* Servlet 4.0
-* JSP 2.3
-* EL 3.0
-* JMS 2.0
-* JTA 1.2
-* JavaMail 1.6
-* Connector 1.7
-* Web Services 1.4
-* JAX-RS 2.1
-* WebSocket 1.1
-* JSON-P 1.1
-* JSON-B 1.0
-* Concurrency Utilities 1.0
-* Batch 1.0
-* Java EE Management 1.1
-* JACC 1.5
-* JASPIC 1.1
-* Java EE Security 1.0
-* JSP Debugging 1.0
-* JSTL 1.2
-* Web Services Metadata 2.1
-* JSF 2.3
-* Common Annotations 1.3
-* Java Persistence 2.2
-* Bean Validation 2.0
-* Managed Beans 1.0
-* Interceptors 1.2
-* Contexts and Dependency Injection for Java
-EE 2.0
-* Dependency Injection for Java 1.0
+* Jakarta Enterprise Beans 3.2 (except for Jakarta Enterprise Beans entity beans and
+associated Jakarta Enterprise Beans QL, which have been made optional)
+* Jakarta Servlet 4.0
+* Jakarta Server Pages 2.3
+* Jakarta Expression Language 3.0
+* Jakarta Messaging 2.0
+* Jakarta Transactions 1.2
```

Comment:
I think this should be 1.3 https://github.com/eclipse-ee4j/jta-api/blob/1.3.2/pom.xml#L35

Reply:
Thanks. Fixed.